doc/release-process.md

```diff
@@ -24,6 +24,7 @@ Before every major release:
 * Update hardcoded [seeds](/contrib/seeds/README.md), see [this pull request](https://github.com/bitcoin/bitcoin/pull/7415) for an example.
 * Update [`BLOCK_CHAIN_SIZE`](/src/qt/intro.cpp) to the current size plus some overhead.
 * Update `src/chainparams.cpp` chainTxData with statistics about the transaction count and rate.
+* Update version of `contrib/gitian-descriptors/*.yml`: usually one'd want to do this on master after branching off the release - but be sure to at least do it before a new major release
```

Comment:
Possible to update this text while you're editing it for less ambiguity? (maybe this should be a separate PR)
suggestion:
> Update version of `contrib/gitian-descriptors/*.yml` on master after branching off the release before any new major release. Updating for minor releases is optional